src/middlewared/middlewared/plugins/disk.py

```diff
@@ -830,6 +831,10 @@ def check_clean(self, disk):
         else:
             disk['disk_identifier'] = await self.middleware.call('datastore.insert', 'storage.disk', disk)
 
+        if await self.middleware.call('service.started', 'collectd'):
```

Comment:
I am not understanding why we need to check if the service has been started?

Reply:
To ensure we won't restart collectd that is not started yet when we are doing boot-time `disk.sync_all`